web/Dockerfile

```diff
@@ -23,3 +23,25 @@ RUN npm run build
 WORKDIR /usr/src/server
 EXPOSE 3002
 CMD [ "npm", "start" ]
+
+RUN apt-get update
+RUN apt-get upgrade -y
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get install -y wget cmake build-essential libeigen3-dev libflann-dev \
+  libpcap0.8-dev libvtk6-dev libboost-all-dev
+ 
+RUN mkdir /pcl
+WORKDIR /pcl
+
+RUN wget https://github.com/PointCloudLibrary/pcl/archive/pcl-1.9.1.tar.gz
+RUN tar -xvzf pcl-1.9.1.tar.gz
+WORKDIR pcl-pcl-1.9.1
+WORKDIR build
+RUN cmake -DCMAKE_BUILD_TYPE=Release ..
+RUN make -j8
+RUN make -j8 install
+
+WORKDIR /
+RUN rm -rf pcl
```

Comment:
will this be changing much? if it's going to take awhile it should go at the top (just after the FROM)

also I think the node CMD line for running the server has to go at the end anyway

Reply:
I'll move these commands to the top of the file and make sure the CMD is the last line.